src/reducers/index.js

```diff
@@ -354,3 +356,47 @@ export function isFetchingTriggerTemplates(state) {
     state.triggerTemplates
   );
 }
+
+export function getTriggerBindings(
+  state,
+  { filters, namespace = getSelectedNamespace(state) } = {}
+) {
+  const bindings = triggerBindingsSelectors.getTriggerBindings(
+    state.triggerBindings,
+    namespace
+  );
+  return bindings.filter(binding => {
+    return filters.every(filter => {
+      const [filterKey, filterValue] = filter.split('=');
+      return (
+        binding.metadata.labels &&
+        filterKey &&
+        filterValue &&
+        binding.metadata.labels[filterKey] === filterValue
+      );
+    });
+  });
+}
+
+export function getTriggerBinding(
+  state,
+  { name, namespace = getSelectedNamespace(state) }
+) {
+  return triggerBindingsSelectors.getTriggerTemplate(
```

Comment:
@carlos-logro I get a warning about this running locally, should it be `getTriggerBinding`?

Reply:
yes you're correct! ill correct this with the EventListener page I'm about to submit